app/models/foram.rb

```diff
@@ -32,6 +32,16 @@ def self.all_attribute_names(user: nil, camelize: false)
     camelize ? names : names.map(&:underscore)
   end
 
+  def self.filterable_attribute_names(user: nil, camelize: false, only_numeric: false)
+    types = [Array, Numeric]
+    types << TrueClass << FalseClass unless only_numeric
```

Comment:
Fajne, choć na Ruby to nie wygląda. :)

```ruby
types += [TrueClass, FalseClass] unless only_numeric
```
albo, jeśli chcemy wycisnąć te 1ms więcej zużywając 0.01kb mniej pamięci
```ruby
types.concat [TrueClass, FalseClass] unless only_numeric
```
Chyba że jest precedens na takie wężyki gdzieś :) osobiście się nie spotkałem